lib/curly/compiler.rb

```diff
@@ -107,7 +107,7 @@ def compile_collection(block)
         items = Array(#{method_call})
         items.each_with_index do |item, index|
           options = options.merge("#{name}" => item, "#{counter}" => index + 1)
-          presenter = #{item_presenter_class}.new(self, options)
+          presenter = ::#{item_presenter_class}.new(self, options)
```

Comment:
Does this work properly with nested presenter classes, e.g. `PostPresenter::CommentPresenter`? The output should be `::PostPresenter::CommentPresenter`, not `::CommentPresenter`. Just wanna make sure.